### PR TITLE
TDP-4003 update mute hold

### PR DESCRIPTION
**Pull request by rmello-daitan**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-update-mute-hold.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-update-mute-hold.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-update-mute-hold.s3-website-us-east-1.amazonaws.com/

File: SUMMARY.md [hunk 1]
```diff
@@ -122,6 +122,8 @@
 		* [POST Transcribe recording](voice/methods/recordings/postCallsCallIdRecordingsRecordingIdTranscription.md)
 		* [GET Download Transcription](voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md)
 		* [DELETE Delete Transcription](voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md)
+	* [Conferences](voice/methods/conferences/about.md)
+	    * [PUT Update Conference Member](voice/methods/conferences/putConferenceMember.md)
```
Comment (muesono-daitan) [inline SUMMARY.md]: The other files are named after their uri. So to follow the same pattern this should be named `putConferencesConferenceIdMembersMemberId.md`.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-update-mute-hold.s3-website-us-east-1.amazonaws.com/

**Pull request closed without merging**